ios release is failing

### What
- ios release is failing
```
Run ci/pub_upgrade.sh
Running 'flutter pub upgrade' in /Users/runner/work/smooth-app/smooth-app/packages/smooth_app
Resolving dependencies...
The current Dart SDK version is 2.19.6.

Because smooth_app requires SDK version >=3.0.0 <4.0.0, version solving failed.
pub finished with exit code 1
Error: Process completed with exit code 1.
```

--- a/.github/workflows/ios-release-to-org-openfoodfacts-scanner.yml
+++ b/.github/workflows/ios-release-to-org-openfoodfacts-scanner.yml
@@ -68,9 +68,9 @@ jobs:
       - name: Setup Flutter
         uses: subosito/flutter-action@v2
         with:
-          #channel: stable
+          channel: stable
           cache: true
-          flutter-version: '3.7.12'
+          #flutter-version: '3.10.4'
           cache-key: current-stable
 
       - name: Flutter version    

